cmd/admin-main.go

```diff
@@ -43,7 +43,7 @@ var adminCmd = cli.Command{
 }
 
 func mainAdminBefore(ctx *cli.Context) error {
-	color.Yellow("\t *** Warning:Experiment feature. Not Ready for Production ***")
+	color.Yellow("\t *** mc admin heal is EXPERIMENTAL ***")
```

Comment:
My understanding is we want this when they type mc admin itself. We do have to fix this error message asap. Right now the typo is egregious.